torch/_functorch/aot_autograd.py

```diff
@@ -93,19 +93,6 @@ def strict_zip(*iterables, strict=True, **kwargs):
     )
 )
 
-pytree._register_pytree_node(
-    immutable_collections.immutable_list,
-    lambda x: (list(x), None),
-    lambda x, c: immutable_collections.immutable_list(x),
-)
-pytree._register_pytree_node(
-    immutable_collections.immutable_dict,
-    lambda x: (list(x.values()), list(x.keys())),
-    lambda x, c: immutable_collections.immutable_dict(
-        dict(zip(c, x))
-    ),
-)
```

Comment:
Were these just unused? I don't see them being moved elsewhere

Reply:
They are defined in fx immutable_collections.py